moduleroot/Gemfile

```diff
@@ -20,7 +22,7 @@ group <%= group %> do
 <% maxlen = gems.map! do |gem| -%>
 <%            { -%>
 <%              'gem'     => gem['gem'], -%>
-<%              'version' => gem['version'], -%>
+<%              'version' => gem['version'].to_s =~ /(^$)|(location_for)/ ? gem['version'] : "'#{gem['version']}'", -%>
```

Comment:
I'd almost prefer using a different key in the config here, instead of :version, so that the templatae renders the ENV , instead of passing through random ruby code.

so in the config_defaults:

```
- gem: beaker-rspec
  from_env: BEAKER_RSPEC_VERSION
```

and here in the Gemfile.erb add `location_for(ENV["<%= gem['from_env'] %>"]` instead.